src/openforms/js/components/admin/form_design/variables/constants.js

```diff
@@ -0,0 +1,26 @@
+const COMPONENT_DATATYPES = {
+    'date': 'datetime',
+    'time': 'time',
+    'file': 'object',
+    'currency': 'float',
+    'number': 'float',
+    'checkbox': 'boolean',
+    'selectboxes': 'object',
+    'npFamilyMembers': 'object',
+    'map': 'array'
+};
+
+// Component types that don't need to have an associated FormVariable
+const NO_VARIABLE_COMPONENT = ['content', 'fieldset', 'column'];
```

Comment:
This might be done a bit more robustly by leveraging Formio's `isLayoutComponent` check: https://help.form.io/developers/javascript-utilities#islayoutcomponent-component